python.pri

```diff
@@ -0,0 +1,35 @@
+win32 {
+	PYTHON=python
+} else {
+	PYTHON=$$system(which python 2>/dev/null)
```

Comment:
Yes, that's the whole point.

I'm sorry that wasn't in the commit message, it was meant to be.

Python from OpenBSD's ports tree only has "/usr/local/bin/python2.7" when installed.

This thing was more of a catch-all for other systems with the same behavior.